core/model/models/network-config.js

```diff
@@ -115,7 +115,7 @@ exports.NetworkConfig = AbstractModel.specialize({
             iconComponentModule: {
                 id: 'ui/icons/network-configuration.reel'
             },
-            nameExpression: "'Network Configuration'"
```

Comment:
Did you change it manually here?
If you update `core/model/user-interface-descriptors/network-config-user-interface-descriptor.mjson` or any user-interface-descriptors and run the tool it will generate for you that data.